<launch>
    <node name="utm_odometry_node" pkg="gps_common" type="utm_odometry_node" output="screen">
        <remap from="odom" to="gps_odom"/>
    </node>
    <node name="wifi_data_collector" pkg="wifi_localization" type="wifi_data_collector" output="screen">
        <param name="threshold" type="double" value="1.0" />
        <param name="store_data_separately" type="bool" value="true" />
        <param name="user_input" type="bool" value="false" />
        <param name="record_only_stopped" type="bool" value="false" />
        <param name="record_wifi_signals" type="bool" value="true" />
        <param name="path_to_csv" type="string" value="" />
        <param name="use_gps" type="bool" value="true" />
        <param name="initial_gps_x" type="double" value="0.0"/>
        <param name="initial_gps_y" type="double" value="0.0"/>
    </node>
</launch>